src/components/Item/PrintTitlePageItem/Item.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types'
 import React from 'react'
 import { connect } from 'react-redux'
 import { sGetIsEditing } from '../../../reducers/editDashboard.js'
-import { sGetNamedItemFilters } from '../../../reducers/itemFilters.js'
+import { msGetNamedItemFilters } from '../../../reducers/itemFilters.js'
```

Comment:
What is the "ms" convention? I haven't seen it before.

Reply:
True it is a new convention I suggested and introduced for "memoized selectors". I discussed it with Jan the other day in [this Slack thread](https://dhis2.slack.com/archives/G3TL0J145/p1738768600819609). Let me know it the convention and the motivation behind it makes sense to you. And also if we need to document this somewhere.

Reply:
It's fine, just needed to know what it meant. Probably something we should share in some sort of conventions document or something.